### PR TITLE
devenv: relax minimum version

### PR DESCRIPTION
**Pull request by joshuarli**
f/u to https://github.com/getsentry/sentry/pull/81274

### CODE REVIEW

File: devenv/sync.py [hunk 1]
```diff
@@ -72,13 +72,18 @@ def run_procs(
...
     parsed_version = tuple(map(int, version.split(".")))
     parsed_minimum_version = tuple(map(int, minimum_version.split(".")))
 
-    if parsed_version < parsed_minimum_version:
+    return parsed_version >= parsed_minimum_version
+
+
+def main(context: dict[str, str]) -> int:
+    minimum_version = "1.13.0"
+    if not check_minimum_version(minimum_version):
```
Comment (asottile-sentry) [inline devenv/sync.py:86]: aren't we able to remove this now that `devenv` itself does this check too?

**Review by asottile-sentry — APPROVED** [review]

File: devenv/sync.py (hunk 1 above)
Comment (joshuarli) [reply devenv/sync.py:86]: there's no way of knowing if everyone is synced to at least 1.13.0 yet - don't want to risk more breakage

Comment (asottile-sentry) [reply devenv/sync.py:86]: makes sense -- I guess we can wait a few weeks and then remove it 👍

Comment (joshuarli) [reply devenv/sync.py:86]: yup thats the plan!

**Pull request merged**